benchmarks/testminer.py

```diff
@@ -694,3 +700,73 @@ def get_result_data(self, test_job_id):
             return (None, None)
         return (db_attachments[0][0], base64.b64decode(db_attachments[0][1]))
 
+
+class AndroidMultinodeBenchmarkResults(LavaTestSystem):
+    def __init__(self, *args):
+        self.host_test_id = "lava-android-benchmark-host"
+        super(AndroidMultinodeBenchmarkResults, self).__init__(*args)
+
+    def get_test_job_results(self, test_job_id):
+        if self.host_test_id == None:
+            return []
+        logger.debug("Multinode measurement parsing from: {0}".format(self.host_test_id))
+
+        status = self.call_xmlrpc('scheduler.job_status', test_job_id)
+
+        if not ('bundle_sha1' in status and status['bundle_sha1']):
+            return []
+
+        test_result_list = []
```

Comment:
this variable never changes and is only used below for returning an empty result set